apps/nextjs/src/utils/testHelpers/consumeStream.ts

```diff
@@ -14,3 +14,12 @@ export async function consumeStream(
 
   return result;
 }
+
+export function extractStreamMessage(streamedText: string) {
+  const content = streamedText.match(/0:"(.*)"/);
```

Comment:
Could this potentially accidentally match an escaped "? 

Should it be `/0:"(.*)"^/` ?

Reply:
I *think* it would be greedy so it should capture the largest possible string, but I agree that it's ambiguous. I can match for the start as well

Reply:
My attempt at this wasn't working, and I'm having some local troubles running the test quite. I'll leave it for now given that this is test code rather the production code, and that it seemed to be running ok on main until it was reverted